app/templates/suppliers/edit_supplier_organisation_size.html

```diff
@@ -27,9 +27,7 @@
 <div class="single-question-page">
   <div class="grid-row">
     <div class="column-two-thirds">
-      {% with heading = "What size is your organisation?", smaller = True %}
-        {% include "toolkit/page-heading.html" %}
-      {% endwith %}
+      <h1 class="govuk-heading-l">$1</h1>
```

Comment:
Should this be `"What size is your organisation?"`

Reply:
```suggestion
      <h1 class="govuk-heading-l">What size is your organisation?</h1>
```